var/spack/repos/builtin/packages/py-dill/package.py

```diff
@@ -11,7 +11,8 @@ class PyDill(PythonPackage):
 
     homepage = "https://github.com/uqfoundation/dill"
     url      = "https://pypi.io/packages/source/d/dill/dill-0.2.6.zip"
```

Comment:
Can you change this extension to `dill-0.2.7.tar.gz`? That will allow `spack checksum` to pick up new versions in the future.

Reply:
Ah, I think I see why it's needed.